docs/contributing.md

```diff
@@ -76,8 +76,15 @@ scripts which maximally automate the process:
   usage.
   
 - **Testing**
+  For the test suite to run you need a current version of Chrome installed.
   Use `node ./run test` run both unit and web-based visual test.
   
+  - *Note for Windows users*:
+    there is a [known issue with wasm-pack](https://github.com/rustwasm/wasm-pack/issues/611) using the
```

Comment:
The mentioned issue is closed. Does it mean it was fixed or considered not to be an issue?

Reply:
I think they fixed an issue in their tests, and I can run those just fine. 
Our test suite fails, however, if I don't have a manually installed chrome driver. 
The `wasm-pack` version on my system seems current though (`0.8.1`).

I think this is how we solve it in our CI: https://github.com/luna/ide/commit/70f082cdd5bda579990af82819144567e2bbc5f6
This also seems to download and install the working version of the chrome driver binary.

From the issue in the wasm-pack repo, it seems that both @farmaazon  and @mwu-tow  also ran into this at some point.

Maybe it is an issue with our test suite instead?

Reply:
I might be wrong but I feel its because e we did not update `wasm-pack` on our side and we cant now cause we are patching it. Is that the case @MichaelMauderer @farmaazon ?